bin/compare-factory-config.py

```diff
@@ -237,6 +234,9 @@ def run(argv):
 
     shutil.rmtree(temp_dir, onerror=remove_readonly)  # file cleanup
```

Comment:
Where treating the user's input as `temp_dir` bites us! I bet that you'd be pretty mad if a program ate all of the date that you provided to it.